pkg/kubelet/container/runtime.go

```diff
@@ -52,6 +52,10 @@ type Runtime interface {
 	// TODO(yifan): Pull/Remove images
 }
 
+type ContainerRunner interface {
```

Comment:
@vmarmol I was considering that but I couldn't find an object that implemented the interface or a usage of the interface anywhere. Did I miss something?